src/schedule/components/ScheduleCard.tsx

```diff
@@ -32,56 +58,66 @@ const ScheduleCard = ({
   durationStart,
   durationEnd,
   createdAt,
+  participateNum,
+  participateCapacity,
+  recruitStart,
+  recruitEnd,
   like,
   comment,
   marked,
+  approvalStatus,
   onClickDelete,
 }: ScheduleCardProps) => {
-  const [isDelete, setIsDelete] = useState(false);
-  const handleDelete = () => {
-    setIsDelete((prev) => !prev);
+  const { openModal, closeModal } = useModal();
+  const [isDeleteToggle, setIsDeleteToggle] = useState(false);
+  const handleDeleteToggle = () => {
+    setIsDeleteToggle((prev) => !prev);
   };
 
-  return (
+  const handleModal = async (id: number) => {
+    const res = await getApplicantsList(id);
+
+    openModal<RecruitManageProps>({
+      contentId: "RecruitManage",
+      scheduleTitle: title,
+      applicants: res.data,
+      participateCapacity: participateCapacity ?? 0,
+      noCallback: () => {
+        closeModal();
+      },
+    });
+  };
+
+  return cardType === "complete" ? (
     <div
-      className={`w-[260px] relative border-gray-300 border box-content transition-transform hover:-translate-y-1 ${
-        isDelete ? "" : "cursor-pointer"
+      className={`w-[260px] relative border-gray-300 border box-content transition-transform hover:-translate-y-1 m-auto ${
+        isDeleteToggle ? "" : "cursor-pointer"
       }`}
     >
-      {isDelete && (
-        <div className="w-[260px] top-0 bottom-0 bg-black bg-opacity-60 absolute z-20">
-          <div className="flex">
-            <div
-              className="relative left-2 mt-4 w-[67px] h-[25px] bg-white rounded-[3px] cursor-pointer"
-              onClick={handleDelete}
-            >
-              <div className="p-1 text-[12px] font-semibold text-center text-zinc-800">
-                취소
-              </div>
-            </div>
-            <div
-              className="relative -right-28 mt-4 w-[67px] h-[25px] bg-white rounded-[3px] cursor-pointer"
-              onClick={() => {
-                handleDelete();
-                onClickDelete(idx);
-              }}
-            >
-              <div className="p-1 text-[12px] font-semibold text-center text-zinc-800">
-                삭제하기
-              </div>
-            </div>
-          </div>
-        </div>
-      )}
-      {/* 사진 및 테마 */}
       <div
         className={`w-[260px] h-[225px] bg-stone-300 border-b border-zinc-400 relatvie`}
```

Comment:
[Lv3] 이부분 클래스네임 오타나있습니다!
```suggestion
        className={`w-[260px] h-[225px] bg-stone-300 border-b border-zinc-400 relative`}
```